### ISSUE TITLE
Make "whole-genome" view optional

### ISSUE THREAD

**Issue by DennisSchmitz** [igvteam/igv.js#1012]
Hi,

Firstly, thank you very much for IGVjs, I really like it and incorporated it into our workflow. We use to to visualize clinical metagenomics data and assess pathogen scaffolds (e.g. important SNPs, biases, errors, etc.). 

However, from version 2.2.16 onward, when trying to open datasets I receive timeout errors from the browser (Chrome). When downgraded to an earlier version (2.2.15) the dataset opens up just fine. The release notes of version 2.2.16 mention:

> Display of whole genome view is now limited to tracks with < 100,000 features.

I'm guessing it has to do with that. Please correct me if I'm wrong, but I assume that the whole genome view is the overview of all "chromosomes" in a sample/dataset as shown [in this example](https://igv.org/web/release/2.3.3/examples/igvjs.html). In my use-case, this would show >10.000 scaffolds.

I'm not interested in that overview since it's not meaningful in a metagenomics use-case. Furthermore, rendering it is quite a burden on the browser. Is there a way that I can stop the rendering of the whole genome view? Or maybe a configuration to open the first chromosome instead of the whole genome view?

I couldn't find a solution in the documentation and any help would be greatly appreciated :)

**Comment by jrobinso** [comment]
Hi,  apologies for that.  Yes an all sequence view doesn't make sense in
that case,  IGV desktop has some heuristic for this,  I will look into
incorporating it.   In the meantime you can jump to the first sequence
using the "locus" configuration parameter described here:
https://github.com/igvteam/igv.js/wiki/Browser-Creation.   In your case the
locus would be the name of your first scaffold.

**Comment by jrobinso** [comment]
If you don't mind I'm going to rename this issue and leave it open.

**Comment by DennisSchmitz** [comment]
Hi, thanks for the quick reply. 

I'll play around with the locus option. But at first glance the documentation mentions a locus string, so no positional parameter (e.g. "1") allowed? Since my scaffolds have random names I'll have to create a workaround then.

**Comment by jrobinso** [comment]
I just pushed a flag to make whole genome (all sequences) view optional.  It will be in the next point release, example follow

```
            reference: {
                id: "myCustomGenome",
                fastaURL: "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg19/hg19.fasta",
                indexURL: "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg19/hg19.fasta.fai",
                cytobandURL: "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg19/cytoBand.txt",
                wholeGenomeView: false
            }
```

**Comment by DennisSchmitz** [comment]
Perfect, thank you!

**Comment by jrobinso** [comment]
This is released (version 2.3.5).
